modules/infinite_ad_entity/infinite_ad_entity.module

```diff
@@ -121,13 +121,15 @@ function _infinite_ad_entity_filter_main_channel(&$vars) {
       $targeting_channel = $targeting_collection->get('channel');
       if(isset($targeting_channel) && is_array($targeting_channel) && $entity->hasField('field_channel')) {
         $channel_term = $entity->get('field_channel')->referencedEntities()[0];
-        $ad_context_item = $channel_term->get('field_ad_context')->first();
-        if(isset($ad_context_item)) {
-          $ad_context = $ad_context_item->getValue();
-          $channel_term_channel = NestedArray::getValue($ad_context, ['context', 'context_settings', 'targeting', 'targeting', 'channel']);
-          if(isset($channel_term_channel)) {
-            $targeting_collection->set('channel', $channel_term_channel);
-            $vars['targeting'] = $targeting_collection->toJson();
```

Comment:
added comment to https://jira.burda.com/browse/INREL-4226

Reply:
This is a bugfix ticket